SPECS/xcp-ng-generic-lib.spec

```diff
@@ -29,11 +30,30 @@ make
 %install
 cd build
 %make_install
+# produce debuginfo stuff: symbols and sources
+%__debug_install_post
 
 %files
 %license LICENSE
 %{_libdir}/libxcp-ng-generic.so.1
 %{_libdir}/libxcp-ng-generic.so.%{version}
+/usr/lib/debug/.build-id/*
+/usr/lib/debug/usr/lib64/*
+
+
+%package debuginfo
+Summary: Debug information for package %{name}
+AutoReqProv: 0
```

Comment:
Out of curiosity, why?

Reply:
Because I've copied the definition of the %debug_package macro. I haven't checked what it does but I can suppose, from the name, that it won't try to add automatically Requires and Provides to the generated package as it does normally.